packages/data-explorer-ui/src/components/common/Banner/banner.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Typography } from "@mui/material";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { CallToActionButton } from "../Button/components/CallToActionButton/callToActionButton";
+import { Banner } from "./banner";
+import { BannerPrimary } from "./banner.styles";
+
+export default {
+  argTypes: {
+    children: { control: { disabled: true } },
+  },
+  component: Banner,
+  title: "Components/Common/Banner",
+} as ComponentMeta<typeof Banner>;
+
+const BannerPrimaryTemplate: ComponentStory<typeof Banner> = (args) => (
+  <Box sx={{ minWidth: 300 }}>
```

Comment:
We can now remove the Box around the template (because we would like the banner to use the full width of the viewport.